hips/hip-0001.md

```diff
@@ -0,0 +1,427 @@
+   They may propose an implementation, but not to Helm's codebase; they often
+   require community consensus. Unlike informational proposals, they are more
+   than recommendations, and users are typically not free to ignore them.
+   Examples include procedures, guidelines, changes to the decision-making
+   process, and changes to the tools or environment used in Helm development.
+
+## HIP workflow
+
+### Start with an idea
+
+The proposal process begins with a new idea for the Helm project. It is highly
+recommended that a single proposal contain a single key proposal or new idea.
+Small enhancements or patches often don't need a proposal and can be injected
+into a project's development workflow with a patch submission to the project's
+issue tracker. The more focused the proposal, the more successful it tends to
+be. As per Helm's governance, the project maintainers reserve the right to
+reject proposals if they appear too unfocused, too broad, or incomplete. If in
+doubt, split your proposal into several well-focused ones, or explain your
+proposal more clearly.
+
+Each proposal must have a champion -- someone who writes the proposal using the
+style and format described below, shepherds the discussions in the appropriate
+communication channels, and attempts to build community consensus around the
+idea. The proposal champion (a.k.a. author) should first attempt to ascertain
+whether the idea is proposal-able. Posting to the [cncf-helm][] mailing list is
+the best way to go about this.
+
+Vetting an idea publicly before going as far as writing a proposal is meant to
+save the potential author time. Many ideas have been brought forward for
+changing Helm that have been rejected for various reasons. Asking the Helm
+community first if an idea is original helps prevent wasting time on something
+that is guaranteed to be rejected based on prior discussions (searching the
+internet does not always do the trick). It also helps to make sure the idea is
+**applicable to the entire community and not just the author**. Just because an
+idea sounds good to the author does not mean it will work for most Helm users in
+most areas where Helm is used.
+
+Once the champion has asked the Helm community as to whether an idea has any
+chance of acceptance, a summary of the proposal should be presented to
+[cncf-helm][]. This gives the author a chance to flesh out the proposal as a
+high quality, well-formatted proposal, and to address initial concerns about the
+proposal.
+
+### Submitting a HIP
+
+Following a discussion on [cncf-helm][], the proposal should be submitted as a
+proposal via a [GitHub pull request][pr]. The proposal must be written in
+proposal style as described below, else it will fail review immediately
+(although minor errors may be corrected by the maintainers).
+
+The standard HIP workflow is:
+
+- You, as the proposal author, fork the helm/community repository and create a
+  new proposal
```

Comment:
Suggestion:

```
- The proposal must be named `hip-XXXX.md` where `XXXX` is the latest HIP in the repo plus 1 
   and placed in `hips/`. E.g. `hips/hip-0002.md`
```

(IMO not an issue, but two PRs may conflict due to the same number)